active/TreeView/TreeViewDragItemsCompletedNewParent.md

```diff
@@ -0,0 +1,43 @@
+# Background
+
+TreeView has built-in support for drag&drop reorder but currently there is no easy way to figure out where the item is being dropped. 
+
+# Description
+
+TreeView's data usually has hierarchical structure by nature and "index" doesn't work very well in this scenario. Parent + SubIndex is a more accurate way to locate items in TreeView. 
+
+A `NewParent` property in TreeViewDragItemsCompletedEventArgs should be sufficient to represent the drop location. To get SubIndex, developers can do NewParent.Children.IndexOf().
+
+# Examples
+
+```C#
+private void TreeView_DragItemsCompleted(MUXC.TreeView sender, MUXC.TreeViewDragItemsCompletedEventArgs args)
+{
+    var parent = args.NewParent;
+}
+```
+
+# API Details
+
+```
+[webhosthidden]
+runtimeclass TreeViewDragItemsCompletedEventArgs
+{
+    ...
+
+    Object NewParent{ get; };

```

Comment:
Not clear if "parent" is an item or a container (or a node). Suggest renaming to NewParentItem